### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maniaque**

### CODE REVIEW

**Comment by truephelle** [comment]
need `sudo make`

**Comment by maniaque** [comment]
Nope, there is something strange with software with `sudo make`, but most users have passwordless sudo on desktop machines :)

File: README.md [hunk 1]
```diff
@@ -4,4 +4,5 @@ Kaspersky leak 2018
 
 # Build
 
-    git clone https://github.com/alukatsky/kaspersky && cd kaspersky && make
+    git clone https://github.com/alukatsky/kaspersky && cd kaspersky
+    ./configure && make
```
Comment (gabriel-fallen) [inline README.md]: `./configure && make && sudo make install`?
Or without `sudo`.

Comment (maniaque) [reply README.md]: Usually `make install` goes with superuser privileges, but original command line was without any install :)

Comment (gabriel-fallen) [reply README.md]: Original command was without any `./configure` so it's not an argument.
Should I send another PR?

**Pull request merged**